src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js

```diff
@@ -258,18 +253,15 @@ const CardSelectedWerkingsgebied = ({
 }
 
 /**
- * Function that renders the WerkingsgebiedPopup component, which is used in the FormFieldWerkingsgebiedKoppeling component, using the imported PopupContainer component, to display the component with an animated entry and closure.
- * This component displays a title and description that instructs the user to select a werkingsgebied that the user wants connected
- * and the list of werkingsgebieden containing a clickable div to set the setInParent variable to the gebied.UUID value and a background image of each werkingsgebied.
+ * Function that renders the WerkingsgebiedPopup component displaying a popup containing a map and a option for the user to connect werkingsgebieden.
  *
- * The component gets the information needed for the werkingsgebieden by using the getAndSetWerkingsgebieden function and for the background image the getImageUrl function.
  *
  * @function
  *
- * @param {boolean} show - Parameter if set true shows the PopupContainer component.
- * @param {boolean} close - Parameter if set true hides (closes) the PopupContainer component.
- * @param {object} setWerkingsgebiedInParentState - Parameter that is used to give value to the target variable and give it to the parent state.
- * @param {object} dataObjectProperty - Parameters value used in the condition statement and part of the value that is set within the setWerkingsGebiedInParentState object.
+ * @param {boolean} show - Parameter that is used to show the PopupContainer component.
+ * @param {boolean} close - Parameter that is called to close the PopupContainer component.
+ * @param {object} setWerkingsgebiedInParentState - Function to edit parent state.
```

Comment:
typeof function

Reply:
I've changed the typeOf from object ==> function for the setWerkingsgebiedInParentState parameter.